crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -127,6 +127,20 @@ pub struct SetAddressCommand {
     pub input: ParsedValue<SuiExtraValueArgs>,
 }
 
+#[derive(Debug, clap::Parser)]
+pub struct AdvanceCheckpointCommand {
+    #[clap(long = "url")]
+    pub url: String,
```

Comment:
what are url and port for

Reply:
copy pasta from breaking up a PR which would let us host the simulator. Fixed